java/client/test/org/openqa/selenium/chrome/ChromeDriverTests.java

```diff
@@ -19,15 +19,104 @@
 
 import org.junit.runner.RunWith;
 import org.junit.runners.Suite;
-import org.openqa.selenium.StandardSeleniumTests;
-import org.openqa.selenium.devtools.DevToolsTests;
+import org.junit.runners.model.InitializationError;
 
-
-@RunWith(Suite.class)
-@Suite.SuiteClasses({
-    StandardSeleniumTests.class,
-    ChromeOptionsFunctionalTest.class,
-    DevToolsTests.class
-})
+@RunWith(ChromeDriverTests.ChromeTestRunner.class)
 public class ChromeDriverTests {
-}
+  /* Use a custom suite to enable use of Class.forName, which is not allowed
+      by the annotations. This isolates tests run by ChromeDriver from changes
+      in the build tools. */ 
+  public static class ChromeTestRunner extends Suite {
```

Comment:
Based on the previous comment from @shs96c https://github.com/SeleniumHQ/selenium/pull/7756; seems like we don't need this file (ChromeDriverTests.java) and can just specify a java_selenium_test_suite block in java/client/test/org/openqa/selenium/chrome/BUILD.bazel. Is that correct? We are not very familiar with the bazel build.

Reply:
That's correct. We use bazel tags for each test too, so you can find and run all the chrome-specific tests using: 

`bazel test --test_tag_filters=chrome //java/...`

Reply:
Thanks, I guess we can close this one and use bazel test --test_tag_filters=chrome //java/... for test.
We used to make the build in Selenium and copied over all the jars for chrome test and run it on our side.